Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerListModel.hxx

```diff
@@ -69,6 +69,7 @@ namespace AzToolsFramework
             ColumnName,                 //!< Entity name
             ColumnVisibilityToggle,     //!< Visibility Icons
             ColumnLockToggle,           //!< Lock Icons
+            ColumnSpacing,              //!< Spacing to allow for drag select
```

Comment:
nit/question: Is it worth adding this to the end just in case the numbering might be important elsewhere? Might not matter but having it after `ColumnSortIndex` might be safer?

Reply:
The Outliner automatically hides the last column, which is the sort index, so unluckily ordering is important. All the rest of the code uses the ENUM so it just catches up nicely, it's just that part.